guides/DevOps/Laravel.md

```diff
@@ -0,0 +1,42 @@
+## The PHP Framework For Web Artisans
+
+[**Laravel**][1] is a free, open-source and considered as one of the most popular [PHP][2] web framework with expressive syntax. Laravel attempts to take the pain out of development by easing common tasks used in the majority of web projects! It provides great tools following [MVC][3] architectural pattern right out of the window to help us build robust web application.
+
+![](https://pbiswas101.files.wordpress.com/2018/07/laravel.png?w=521&h=336)
+
+**Taylor Otwell** created this elegant framework with it's first beta being released on June 9, 2011.
+
+## Laravel Framework For Web Development
+
+Whether you are a seasoned PHP developer or just starting out; a basic understanding of PHP is a prerequisite for getting acquainted with this particular framework but of course the important question:
+
+> Why should you care?
+
+Well, here's a great article explaining just that -

```

Comment:
Instead of giving a link, why don't you explain the reasons concisely in the article itself?